searxng_extra/standalone_searx.py

```diff
@@ -160,6 +162,9 @@ def parse_argument(
     parser.add_argument(
         '--timerange', type=str, nargs='?', choices=['day', 'week', 'month', 'year'], help='Filter by time range'
     )
+    parser.add_argument(
+        '--license', type=str, nargs='?', choices=['any', 'public', 'freetouse', 'commercial'], help='Filter by license'
```

Comment:
I think we should define what the choices mean, its not clear to me e.g what "any" stands for .. does it mean *"don't filter by license"* .. engine developers should have a rough idea of what we want to filter with these categories: 'public', 'freetouse', 'commercial'

Reply:
I agree, but we probably should document it somewhere else than in `standalone_searx.py`, right? Possibly at the Engine file section at https://docs.searxng.org/dev/engines/engine_overview.html?

Reply:
> Possibly at the [Passed arguments documentation](https://docs.searxng.org/dev/engines/engine_overview.html#passed-arguments-request)?

Seems a quite good place to document the meaning of the *license categories*

- [searx/templates/simple/filters/license.html](https://github.com/searxng/searxng/pull/3906/files#diff-e258700dc86b956c917ed2fd84b2188bd21fa2e4742ce0f0e4158d1227867045)

.. since it is more a "category of licenses to filter" what the user will select, we should name the field `filter_lic-categ` or similar.  The lic-filter named *all* should not exists, the user selects a license *category* `filter_lic-categ=[public|freetouse|commercial]` or `None`.